pkg/validation/limits.go

```diff
@@ -143,6 +143,10 @@ type Limits struct {
 
 	RulerRemoteWriteConfig map[string]config.RemoteWriteConfig `yaml:"ruler_remote_write_config,omitempty" json:"ruler_remote_write_config,omitempty" doc:"description=Configures global and per-tenant limits for remote write clients. A map with remote client id as key."`
 
+	// TODO(dannyk): possible enhancement is to align this with rule group interval
+	RulerRemoteEvaluationTimeout         time.Duration `yaml:"ruler_remote_evaluation_timeout" json:"ruler_remote_evaluation_timeout" doc:"description=Timeout for a remote rule evaluation. Defaults to the value of 'querier.query-timeout'."`
+	RulerRemoteEvaluationMaxResponseSize int64         `yaml:"ruler_remote_evaluation_max_response_size" json:"ruler_remote_evaluation_max_response_size" doc:"description=Maximum size (in bytes) of the allowable response size from a remote rule evaluation. Set to 0 to disable."`
```

Comment:
Q: I'm failing to find default value for this config (`RulerRemoteEvaluationMaxResponseSize`). Is that intentional? (if not provided, default value will be 0 and meaning ruler remote evaluation will be disabled?)

Looks bit confusing and contradicting with the doc page on how to enable it?

```yaml
ruler:
  evaluation:
    mode: remote
    query_frontend:
      address: dns:///<query-frontend-service>:<grpc-port>
```

This config never mentions anything about setting `RulerRemoteEvaluationMaxResponseSize` when enabling it? Am i missing anything here?

Reply:
It's an `int64` so its default / zero-value will be `0`.
The wording is misleading here; `Set to 0 to disable` means to disable the limit.

The config is not mentioned in that section above because it's a limit.